charlatan/tests/test_testcase.py

```diff
@@ -50,7 +50,7 @@ def test_install_all_fixtures(self):
         self.uninstall_all_fixtures()
 
         fixtures = self.install_all_fixtures()
-        self.assertEqual(len(fixtures), 3)
+        self.assertEqual(len(fixtures), 4)
```

Comment:
Do you think this would be worth changing this so that this doesn't need changed on additions?

```python
self.assertEqual(len(fixtures), len(fixtures_manager.keys())
```